src/containers/Tenant/Query/i18n/en.json

```diff
@@ -1,7 +1,10 @@
 {
   "controls.query-mode-selector_type": "Query type:",
 
-  "tabs.newQuery": "Query",
+  "sections.query": "Query",
```

Comment:
This translations should not be here. The better place is `/Tenant/ObjectGeneral/TenantWorkModes/i18n`
And please remove old translations from AsideNavigation translations.